pkgs/top-level/all-packages.nix

```diff
@@ -9014,7 +9014,7 @@ in
 
   phantomjs = callPackage ../development/tools/phantomjs { };
 
-  phantomjs2 = libsForQt5.callPackage ../development/tools/phantomjs2 { };
+  phantomjs2 = libsForQt5.callPackage ../development/tools/phantomjs2 { }; 
```

Comment:
Please remove the trailing whitespace here when you cleanup the history.

Reply:
I did clean the history, but I'm not sure I did it correctly (I'm not used to squash commits...). Can you confirm that everything is good ?